_config.yml

```diff
@@ -63,7 +63,8 @@ sched:
   hist: "sacct -u yourUsername"
 
 episode_order:
-  - 11-hpc-intro
+  - 10-hpc-intro
+  - 11-connecting
```

Comment:
If you are using numbered episode files, `episode_order` is redundant. This is not a problem exactly, but I wanted to clarify.